packages/api-explorer/src/scenes/MethodScene/MethodScene.tsx

```diff
@@ -115,6 +118,17 @@ export const MethodScene: FC<MethodSceneProps> = ({ api }) => {
     checkRunIt()
   }, [adaptor, setOn])
 
+  useEffect(() => {
+    if (method.responses && errorCode) {
+      const maybe = true
+      if (maybe && docResponsesRef.current) {
```

Comment:
what is `maybe` for here?

Reply:
I think it can be removed. I'll address.